cmdstanpy/stanfit.py

```diff
@@ -134,29 +134,56 @@ def is_optimizing(self) -> bool:
     @property
     def optimized_params_np(self) -> np.array:
         """returns optimized params as numpy array"""
-        self._optimizing_only()
         return self._first_draw
 
     @property
     def optimized_params_pd(self) -> pd.DataFrame:
         """returns optimized params as pandas DataFrame"""
-        self._optimizing_only()
         return pd.DataFrame([self._first_draw], columns=self.column_names)
 
-    @property
-    def optimized_params_dict(self) -> dict:
+    def optimized_params_dict(self) -> OrderedDict:
```

Comment:
prefer the previous implementation.

Reply:
thought about it more - definitely prefer that optimized_params_dict directly reflect parameters in the model, i.e., all elements of a container are distinct params.   please revert to earlier implementation.